libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -341,8 +501,21 @@ def process_response(self, document: Document) -> GraphDocument:
         """
         text = document.page_content
         raw_schema = self.chain.invoke({"input": text})
-        raw_schema = cast(Dict[Any, Any], raw_schema)
-        nodes, relationships = _convert_to_graph_document(raw_schema)
+        if self._function_call:
+            raw_schema = cast(Dict[Any, Any], raw_schema)
+            nodes, relationships = _convert_to_graph_document(raw_schema)
+        else:
+            nodes = []
+            relationships = []
+            parsed_json = self.json_repair.loads(raw_schema.content)
+            for rel in parsed_json:
+                source_node = Node(id=rel["head"], type=rel["head_type"])
+                target_node = Node(id=rel["tail"], type=rel["tail_type"])
+                relationships.append(
```

Comment:
where do we append nodes?

Reply:
We don't because we don't really need them

Reply:
But I added them anyway

Reply:
thx, you never know what people will use them for :)